docs/csharp/roslyn-sdk/tutorials/snippets/how-to-write-csharp-analyzer-code-fix/NuGet.config

```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
```

Comment:
📝 There's a pretty good chance parts of this won't work for people without a NuGet.org line

Reply:
@sharwell This is what the latest template updates have after https://github.com/dotnet/roslyn-sdk/pull/725.

I actually opened an issue https://github.com/dotnet/roslyn-sdk/issues/742 for one case which this wouldn't work.

Reply:
Once we resolve this question, this is ready to merge.

@sharwell, what do you recommend here?

Reply:
@BillWagner Is it possible to address this in a follow-up in case dotnet/roslyn-sdk#742 takes some time?

Reply:
@Youssef1313 @sharwell Yes, I'm happy with that plan. I'll plan on merging later today, unless Sam disagrees.